thorlcr/activities/indexwrite/thindexwrite.cpp

```diff
@@ -159,9 +159,9 @@ class IndexWriteActivityMaster : public CMasterActivity
             checkFormatCrc(this, _f, helper->getFormatCrc(), nullptr, helper->getFormatCrc(), nullptr, true);
```

Comment:
future: could add extra protection against user error:

       if (!isFileKey(_f))
            throw MakeActivityException(this, ENGINEERR_FILE_TYPE_MISMATCH, "Attempting to read flat file as an index: %s", indexFileName.get());